test/e2e/helm_test.go

```diff
@@ -0,0 +1,98 @@
+package e2e
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+	"sigs.k8s.io/e2e-framework/third_party/helm"
+
+	// e2enetwork "k8s.io/kubernetes/test/e2e/framework/network"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+// Actual test suite
+var _ = NFDDescribe(Label("helm"), func() {
+	f := framework.NewDefaultFramework("node-feature-discovery")
+	// To avoid the error "seccompProfile (pod or container must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")"
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline
+
+	manager := helm.New(framework.TestContext.KubeConfig)
+	chart_name := "nfd/node-feature-discovery"
+	release_name := "node-feature-discovery"
+	version := "0.16.3"
+
```

Comment:
We should test local helm charts, instead of released ones, this way we can tests if current changes will break Helm deployment